lib/tasks/peoplefinder/data.rake

```diff
@@ -141,5 +141,37 @@ namespace :peoplefinder do
       end
     end
 
+    namespace :etc do
+      teams_to_migrate = [
+        {from: 'hmp-lewes-do-not-use', to: 'hmp-lewes'},
+        {from: 'hmp-coldingley-do-not-use', to: 'hmp-coldingley'},
+        {from: 'south-central-prisons-group-hmp-huntercombe-do-not-use-2',
+            to: 'south-central-prisons-group-hmp-huntercombe-2'}
+      ]
+
+      desc 'Bulk move people from an old to a new team, preserving existing members in new team, and delete the old team'
+      task :bulk_move_lewes_huntercombe_coldingley => :environment do
+        teams_to_migrate.each do |team|
+          begin
+            puts "Assign #{team[:from]} members to #{team[:to]}"
+            old_team = Group.find_by_slug team[:from]
+            new_team = Group.find_by_slug team[:to]
+            if old_team && new_team
+              puts "Copying #{old_team.people.count} members"
+              new_team.people << old_team.people
+              old_team.people.delete_all
```

Comment:
I presume you've tested this, but just wanted to make sure that doing `new_team.people << old_team.people` means you end up with 2 sets of people, so it is then safe to delete them from old_team.

Reply:
Sorry, missed this! Yep, exactly so, I've just been testing it again locally as a sanity check. 

The many-to-many table `memberships` is what's getting modified here so for example 

```[12] pry(main)> old_team.people.delete_all
  Membership Destroy (43.9ms)  DELETE FROM "memberships" WHERE "memberships"."id" IN (SELECT "memberships"."id" FROM "memberships" LEFT OUTER JOIN "people" ON "people"."id" = "memberships"."person_id" WHERE "memberships"."group_id" = $1 AND "memberships"."person_id" IN ($2, $3, $4, $5, $6, $7, $8, $9, $10) ORDER BY people.surname)  [["group_id", 18], ["person_id", 74], ["person_id", 73], ["person_id", 61], ["person_id", 50], ["person_id", 62], ["person_id", 72], ["person_id", 60], ["person_id", 48], ["person_id", 49]]
=> 9
```